packages/bot-web-ui/src/stores/flyout-help-store.js

```diff
@@ -50,6 +50,30 @@ export default class FlyoutHelpStore {
         }
     };
 
+    getHelpContent = async block_node => {
+        let block_content;
+
+        if (block_node) {
+            const target_blocks = this.xml_list_group[block_node];
+            const block_type = target_blocks[0].getAttribute('type');
+            const help_string_obj = await import(/* webpackChunkName: `[request]` */ '@deriv/bot-skeleton');
+            block_content = help_string_obj.default[block_type];
+        }
+
+        return block_content;
+    };
+
+    getFilledHelpContents = async blocks_Type => {
+        const blocks_content = await Promise.all(blocks_Type.map(block => this.getHelpContent(block)));
+        return blocks_content.map((key, index) => (key != null ? index : null)).filter(value => value != null);
```

Comment:
- Please always use strict comparison operators, i.e. `key != null` should be `key !== null`.
- In this case you could use `key ? index : null`
- You could leave out the last `!= null` as `filter` is ok with a truthy value. e.g. `[...].filter(value => value)`

```suggestion
        return blocks_content.map((key, index) => (key ? index : null)).filter(value => value);
```

Reply:
re "You could leave out the last != null as filter is ok with a truthy value. e.g. [...].filter(value => value)" : The array contains 0, which will consider "false" and it will be ommited in this way, so I use the strict comparison.